documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -2,6 +2,10 @@
 Release Notes
 #############
 
+6.3.19
+======
+* Add the ``trace_partial_file_suffix`` network option. This option will give unfinished trace files a special suffix to indicate they're not complete yet. When the trace file is complete, it is renamed to remove the suffix. `(PR #5330) <https://github.com/apple/foundationdb/pull/5330>`_
```

Comment:
I don't actually have a good handle on our process here, but I think sometimes we've been adding these release notes to 7.0 as well (I've done so, at least). I'm not sure where the line is when we should start dual reporting them, and if you have reason to think this shouldn't be in the 7.0 release notes I'm ok with that.

Reply:
Done